engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/JoinGameScreen.java

```diff
@@ -92,6 +85,9 @@
     @In
     private StorageServiceWorker storageServiceWorker;
 
+    @In
+    private BroadCastServer broadCastServer;
```

Comment:
`BroadCastServer` is just a regular class without any annotations or such. Thus, I don't think it can just be injected like this.